cscs-checks/mch/openacc_cuda_mpi_cppstd.py

```diff
@@ -17,23 +19,41 @@ def __init__(self):
             self.num_tasks_per_node = 12
             self.num_gpus_per_node = 1
             self.build_system.options = ['NVCC_FLAGS="-arch=compute_60"']
+            self.variables = {
+                'MPICH_RDMA_ENABLED_CUDA': '1',
+                'CRAY_CUDA_MPS': '1'
+            }
         elif self.current_system.name == 'kesch':
             self.exclusive_access = True
-            self.modules = ['craype-accel-nvidia35']
+            self.modules = ['cudatoolkit/8.0.61']
             self.num_tasks = 8
             self.num_tasks_per_node = 8
             self.num_gpus_per_node = 8
             self.build_system.options = ['NVCC_FLAGS="-arch=compute_37"']
-            # FIXME: temporary workaround until the mvapich module is fixed;
-            #        'PrgEnv-{pgi,gnu}-c2sm-gpu' will be added later
-            self.valid_prog_environs += ['PrgEnv-cray-c2sm-gpu']
-
-        if self.current_system.name in ['daint', 'dom']:
```

Comment:
Here you are removing the configuration for daint/dom, but these systems are in the `valid_systems`

Reply:
The `if` statement for Dom and Piz Daint has been moved above, on line 17